### PR TITLE
community/php7: fix licenses

### PR DESCRIPTION
**Pull request by andypost**
Checked all libs, follow-up to https://github.com/alpinelinux/aports/pull/4879

### CODE REVIEW

**Comment by andypost** [comment]
Let's use it to fix PHP licenses  for all community packages

**Comment by andypost** [comment]
Does it makes sense to increase `pkgrel`? @jirutka

**Comment by andypost** [comment]
Also commit message needs work

**Comment by ncopa** [comment]
> Does it makes sense to increase `pkgrel`?

Yes. It changes the content so `apk info --license` will display different data. `pkgrel` does not need to be increased when a rebuild produces exactly the same binary (in theory - we still have a way to go to be 100% reproducible)

File: community/php7/APKBUILD [hunk 1]
```diff
@@ -34,7 +35,7 @@ _default_php="yes"
 pkgdesc="The PHP$_suffix language runtime engine"
 url="https://secure.php.net"
 arch="all"
-license="PHP-3.0 BSD LGPL-2.0 MIT Zend"
+license="PHP-3.01 BSD LGPL-2.0-or-later MIT Zend-2.0"
```
Comment (ncopa) [inline community/php7/APKBUILD]: pkgrel needs to be bumped due to this change

**Comment by andypost** [comment]
@ncopa rebased with bump

**Comment by ncopa** [comment]
fixed with 01e1a5c92706c33217712f5f2cd5aede165bb297

**Pull request closed without merging**